pkg/sql/sqlinstance/instancestorage/instancereader_test.go

```diff
@@ -175,6 +175,15 @@ func TestReader(t *testing.T) {
 			}
 			return nil
 		}
+		getInstancesUsingTxnWrapper := func() ([]sqlinstance.InstanceInfo, error) {
```

Comment:
nit: you should take `t` as a parameter to deal with future refactorings and subtests. Also, I don't think the `Wrapper` suffix adds value; just call it `getInstancedUsingTxn`